src/MeshBVHNode.js

```diff
@@ -287,7 +287,12 @@ MeshBVHNode.prototype.intersectsGeometry = ( function () {
 
 			if ( geometry.boundsTree ) {
 
-				function triangleCallback( tri ) {
+				arrayToBox( this.boundingData, obb2 );
+				obb2.matrix.copy( invertedMat );
+				obb2.update();
+
+				cachedMesh.geometry = geometry;
+				const res = geometry.boundsTree.shapecast( cachedMesh, box => obb2.intersectsBox( box ), function triangleCallback( tri ) {
```

Comment:
Let's remove the function name `triangleCallback` because it shouldn't be needed anymore.